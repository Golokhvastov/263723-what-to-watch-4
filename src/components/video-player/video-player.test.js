import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const movie = {
  title: `Тест1`,
  pictureSrc: `test1.jpg`,
  preview: `test2.jpg`,
};

it(`VideoPlayer render correctly`, () => {
  const tree = renderer
    .create(
        <VideoPlayer
          src = {movie.preview}
          posterSrc = {movie.pictureSrc}
          isPlaying = {true}
          width={`100%`}
          height={`100%`}
        />, {
          createNodeMock: () => {
            return {};
          }
        }).toJSON();

  expect(tree).toMatchSnapshot();
});
