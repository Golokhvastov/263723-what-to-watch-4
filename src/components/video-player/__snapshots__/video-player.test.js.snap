// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer render correctly 1`] = `
<video
  height="100%"
  width="100%"
/>
`;
